README.md

```diff
@@ -217,7 +312,13 @@ The actual numeric values are stored in a correspondingly named pkl file.
 You will need to understand a bit of pydra to know how to debug this application for
 now. If the process crashes, the easiest way to restart is to remove the `cache-wf`
 folder first. However, if you are rerunning, you could also remove any `.lock` file
-in the `cache-wf`directory.
+in the `cache-wf`directory. An example was left in the `examples/` folder:
+
+```
+python3 clear_locks.py
+```
+
+A bash script `run_clear_locks.sh` has also been included to run `clear_locks.py` using slurm.
```

Comment:
```suggestion
in the `cache-wf`directory. 
```

removing most of this. with the latest changes in pydra, all one needs to do is remove the lock files in the cache directory. a specialized function to also remove other directories is not required.